tmux-client-173868.log

```diff
@@ -0,0 +1,90 @@
+1640792588.737343 client started (173868): version 3.0a, socket /tmp//tmux-1000/default, protocol 8
```

Comment:
lol

Reply:
I wanted to let you aware of my tech stack :) I'll get rid of it